web-ui/github.ml

```diff
@@ -283,15 +298,18 @@ let repo_handle ~meth ~owner ~name ~repo path =
         respond_error `Internal_server_error reason
     end
   | `POST, ["commit"; hash; "cancel"] ->
+    let job_lookup commit job_info =
+      let variant = job_info.Client.variant in
+      Capability.with_ref (Client.Commit.job_of_variant commit variant) @@ fun job ->
```

Comment:
Using `with_ref` here ensures that `job` is released before `job_lookup` returns, but this isn't what you want in the case where you return the job,